cmd/validators/main.go

```diff
@@ -59,26 +60,32 @@ Available commands: %s
 // commands is a global register of all commands provided by this program. Each
 // command should use flag package to support options and provide help text.
 var commands = map[string]func() error{
-	"list": cmdList,
-	"add":  cmdAdd,
+	"list":            cmdList(os.Stdout),
+	"add":             cmdAdd,
+	"multisig-new":    cmdMultisigNew(os.Stdin),
+	"multisig-sign":   cmdMultisigSign(os.Stdin, os.Stdout),
+	"multisig-submit": cmdMultisigSubmit(os.Stdin),
 }
 
-func cmdList() error {
-	var (
-		tmAddrFl = flag.String("tm", "https://bns.NETWORK.iov.one:443", "Tendermint node address. Use proper NETWORK name.")
-	)
-	flag.Parse()
+func cmdList(out io.Writer) func() error {
+	return func() error {
 
-	info, err := listValidators(*tmAddrFl)
-	if err != nil {
-		return fmt.Errorf("cannot list validators: %s\n", err)
-	}
-	b, err := json.MarshalIndent(info, "", "\t")
-	if err != nil {
-		return fmt.Errorf("cannot serialize: %s", err)
+		var (
+			tmAddrFl = flag.String("tm", "https://bns.NETWORK.iov.one:443", "Tendermint node address. Use proper NETWORK name.")
+		)
+		flag.Parse()
```

Comment:
req: please replace the default with a new `FlagSet` to get rid of all the test flag noise.
https://stackoverflow.com/questions/42243919/using-go-build-but-i-also-see-the-test-flags